gallery/src/index.html

```diff
@@ -7,13 +7,13 @@
 
 -->
 <!DOCTYPE html>
-<html class="nx-html">
+<html class="nx-html nx-html--page-scrolling">
   <head>
     <meta charset="utf-8">
     <title>Sonatype shared component gallery</title>
     <link rel="stylesheet" href="gallery.css"/>
   </head>
-  <body class="nx-body">
+  <body class="nx-body nx-body--page-scrolling">
```

Comment:
Does this class do anything anymore?

Reply:
> Does this class do anything anymore?

Nope: https://github.com/sonatype/sonatype-react-shared-components/pull/38/commits/9b0e42636c2b211c8fc7dd0ae1a47b30ee601a69